swagger.yml

```diff
@@ -2010,6 +2010,36 @@ paths:
         name: path
```

Comment:
Can we also do the JavaScript version nonsense in this PR?  (`make build` and commit)